projects/zetasql/build.sh

```diff
@@ -49,10 +49,10 @@ fi
 )"
 
 # Temporary hack, see https://github.com/google/oss-fuzz/issues/383
-readonly NO_VPTR='--copt=-fno-sanitize=vptr --linkopt=-fno-sanitize=vptr'
+readonly NO_VPTR='--copt=-fno-sanitize=vptr --linkopt=-fno-sanitize=vptr --cxxopt=-fno-sanitize=vptr'
 
-declare FUZZER_PATH="zetasql/fuzzing/simple_evaluator_fuzzer"
-declare FUZZ_TARGET="//zetasql/fuzzing:simple_evaluator_fuzzer"
+declare FUZZER_PATH=$(for path in $(find zetasql/fuzzing -name *_fuzzer.cc); do echo "${path%.*}"; done)
+declare FUZZ_TARGET="$(for fuzzer in ${FUZZER_PATH}; do echo "//$(dirname ${fuzzer}):$(basename ${fuzzer})"; done)"
```

Comment:
So this implies a related naming convention for fuzzer code files & also fuzz targets.  That isn't inherently bad, but it should be documented.  (An alternate option might be to have a naming convention just for the fuzz targets, and use bazel query.)

Reply:
We can use `bazel query` instead which is more error-proof.

Reply:
Do you plan to do this ? Please ping us when this cl is ready for review and CI passed.